core/src/test/java/io/wttech/stubway/response/StubResponseTest.java

```diff
@@ -0,0 +1,36 @@
+package io.wttech.stubway.response;
+
+import io.wttech.stubway.stub.Stub;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+
+public class StubResponseTest {
+
+    @Test
+    public void emptyShouldReturnStubResponseWithEmptyParams() {
+        StubResponse empty = StubResponse.empty();
+        Assertions.assertEquals((Integer) null, empty.getStatusCode());
+        Assertions.assertNull(empty.getInputStream());
```

Comment:
this will also fail. Please use this snipped to parse InputStream
```suggestion
               StubResponse empty = StubResponse.empty();
        Assertions.assertEquals(404, empty.getStatusCode());
        String expected = "{\n"
                + "  \"message\": \"Stub not found\",\n"
                + "  \"statusCode\": 404\n"
                + "}";
        Assertions.assertEquals(expected, IOUtils.toString(empty.getInputStream(), Charset.defaultCharset()));
```